experimental/oak_baremetal_kernel/Cargo.toml

```diff
@@ -21,17 +21,9 @@ lazy_static = { version = "*", features = ["spin_no_std"] }
 linked_list_allocator = { version = "*" }
 log = "*"
 libm = "*"
-oak_remote_attestation = { path = "../../remote_attestation/rust", default-features = false, features = [
```

Comment:
I think the getrandom dependency above needs to also be removed. I can't remember exactly why it was required, but it might have to be moved to experimental/oak_baremetal_app_crosvm/Cargo.toml